### ISSUE TITLE
Balancing suggestion for batch crafting slates

### ISSUE THREAD

**Issue by Vashal0** [WayofTime/BloodMagic#1903]
Talked to Enigmatica 6: Expert people and was recommended to post this suggestion here instead.

Currently if you were to need to automatically craft a large batch of slates, say 1k, the fastest way that I can see is a bit unintuitive.

Crafting 128 demonic slates in stacks of 64 at a time, using this setup:

19x Acceleration Rune
8x Rune of Capacity
24x Rune of Augmented Capacity
16x Charging Rune
28x Displacement Rune
13x Speed Rune

With infinite blood being piped in, took me 19:27, same setup but putting them in one at a time automatically took 10:35, and replacing 2 speed runes with charging, and doing batches of 8 at a time when the charging buffer was full automatically, 5:01.

I don't suggest just making everything super easy, as the more elaborate setup should be more rewarding generally. But it does seem a bit unintuitive that single item crafting is significantly faster than bulk crafting, and that the difference in times is so huge for min maxing. **So I suggest having the lp drain per tick scale in some way with stack size**, or to change the value of speed runes in some way so that with normal bulk crafting it can compete slightly with the alternative. Also I did try another, less generalized altar, replacing all 16 charging runes with speed runes. and the time was just under 12 mins.

All of my testing was done in a creative test world on Enigmatica 6: Expert
____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: 1.16.4-3.1.10-31
- Minecraft: 1.16
- Forge: 36.2.34

**Comment by VT-14** [comment]
> 16x Charging Rune

I suspect that is what is causing the weirdness.

Crafting items 1 at a time is normally a little slower than full stacks because there is a brief delay between inserting the item and the crafting starting.  A single stack of 64 items has a single delay, where as 64 items done 1 at a time would have 64 of those delays.  Other than that delay, crafting occurs at the exact same speed; the LP/t is set by the recipe, and the total LP needed is the same.

However, Charging Runes store LP as Charge to be applied instantly when crafting starts, and then the crafting continues as normal.  I suspect that your combination of Acceleration and Speed Runes means that the Altar can convert LP to Charge faster than the used recipes are able to craft normally.  When the recipe starts crafting the Charge is instantly turned into progress on the recipe, and then it continues with the (in this case slower) normal crafting method.

**Comment by Vashal0** [comment]
Yeah, sorry if I was a bit unclear, but i'm mostly trying to say that with charging runes, other forms of crafting tons of slates are rendered basically obsolete. Which doesn't seem like the intended usage for charging runes. So I'm suggesting having the bulk crafting scale so it keeps up in viability

**Comment by Vashal0** [comment]
this very long youtube video I made showcases what i'm talking about. It does use some other mods in the video, modular routers to move things around and refined storage for auto crafting. But none of those actually change anything. You can skim through it if you like as its mostly just waiting around after the start, and sped up later but the main TLDW is that crafting 2 stacks of demonic slates starting from the E6E equivalent of stone, is roughly 4 times quicker when using charging runes to do all of the crafting (in variable bulk amounts) https://www.youtube.com/watch?v=NDD_RG3hzj0
